### PR TITLE
docs: survey results

### PR DESCRIPTION
**Pull request by babadofar**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

# Checklist:

- [ ] I have mentioned the appropriate type(scope), as referenced [here](https://jenkins-x.io/community/code/#the-commit-message) in the commit message and PR title for the semantic checks to pass.
- [ ] I have signed off the commit, as per instructions mentioned [here](https://jenkins-x.io/community/code/#how-to-sign-your-commits).
- [ ] Any dependent changes have already been merged.

### CODE REVIEW

File: content/en/blog/news/2022-02-15-survey-results.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+This person works with Devops and Software Engineering,
+is using Jenkins X version 3 on Amazon, and working in a company of 1-50 people. 
+The person finds it somewhat difficult to find documentation, and uses the main web site, slack and github when they try to do so.
+The thing they enjoy the most with Jenkins X is how it is an easy way to learn and work with Kubernetes, the git based configuration (gitops) and preview environments.
+But they really think that documention shold be improved. 
+
+This person needs a way to run Jenkins X offline, behind a firewall, also they need support for multi-tenant. Security is of course a concern for many.  A clear governance of Jenkins X needs to be established, 
+
+The person *is* planning to attend Jenkins X Office hours, which is great! See you there next week then!
```
Comment (ankitm123) [inline content/en/blog/news/2022-02-15-survey-results.md:24]: I like the way the short summary is presented :100:

File: content/en/blog/news/2022-02-15-survey-results.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+---
```
Comment (ankitm123) [inline content/en/blog/news/2022-02-15-survey-results.md:1]: why 2 blog posts?

Comment (babadofar) [reply content/en/blog/news/2022-02-15-survey-results.md:1]: I wanted to have one post with summary and overview, and a page with more detailed info. The detail page should only appear as a link in the overview and not as a separate blog with an entry in the menu. But i couldn't figure out how to do that.

Comment (ankitm123) [reply content/en/blog/news/2022-02-15-survey-results.md:1]: You can try `toc_hide: true` and `hide_summary: true` in the markdown front matter. That should atleast hide it from the sidebar. See: https://www.docsy.dev/docs/adding-content/navigation/#section-menu

**Pull request merged**